cmd/admin-handlers_test.go

```diff
@@ -68,7 +68,7 @@ func (c cmdType) toServiceSignal() serviceSignal {
 	return serviceStatus
 }
 
-func serviceSignalReceiver(cmd cmdType, t *testing.T) {
+func mockServiceSignalReceiver(cmd cmdType, t *testing.T) {
```

Comment:
It is mocking the signal receiver code in service.go and named accordingly. It is not initializing anything so it doesn't make sense naming it so.